packages/eip-1193-provider/src/constants.ts

```diff
@@ -0,0 +1,29 @@
+export const PROVIDER_ERROR_MESSAGE = {
+  BLOCK_EXPLORER_URL:
+    "Expected null or array with at least one valid string HTTPS URL 'blockExplorerUrl'. Received: ",
+  INVALID_CHAIN_ID_HEX:
+    "Expected 0x-prefixed, unpadded, non-zero hexadecimal string 'chainId'. Received: ",
+  INVALID_CHAIN_ID_VALUE:
+    "Invalid chain ID '': numerical value greater than max safe value. Received: ",
```

Comment:
I think there's an empty string that should go (`''`) since you're templating the received value after "Received:"